package.json

```diff
@@ -25,6 +25,7 @@
     "@types/codemirror": "0.0.106",
     "@types/diff-match-patch": "^1.0.32",
     "@types/tinymce": "^4.6.0",
+    "codemirror6": "npm:codemirror@^6.0.0",
```

Comment:
Could you move this to dev dependency as its only needed for testing

Reply:
Done